Exercise-Files/Ex11Q1-1.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/ bash
+
+for i in Problem1/*.ref
+	do 
+	../../.././muscle3.8.31_i86win32 -in $i -out $i.aln
+	../../../hmmer-3.1b2-cygwin64/binaries/./hmmbuild $i.hmm $i.aln
+	done 
+cat Problem1/*.fasta >> all-files.fasta
+../../../hmmer-3.1b2-cygwin64/binaries/./hmmsearch Problem1/sigma.ref.hmm all-files.fasta > sigma.hmms
+../../../hmmer-3.1b2-cygwin64/binaries/./hmmsearch Problem1/sporecoat.ref.hmm all-files.fasta > sporecoat.hmms
+../../../hmmer-3.1b2-cygwin64/binaries/./hmmsearch Problem1/transporter.ref.hmm all-files.fasta > transporter.hmms
+	
+ 
```

Comment:
You need to generate a single results file that contains proteome, HMM result and e-value
cat *.hmms | grep -v "#" | awk '{print $1,$3,$5}' | sed -E 's/tr\|[A-Z0-9]+\|[A-Z0-9]+_9//g' > hmmOut.txt

-0.25